src/main/resources/db/local/V99_0__draft_referrals.sql

```diff
@@ -0,0 +1,4 @@
+insert into referral (id, created, completion_deadline)
+values ('F7B4A237-94CC-448F-A2C0-CDFFF17B58B9', '2020-12-07 18:02:01.599803+00', '2021-02-14'),
+       ('71E6EAF6-5899-4A6B-9651-1AA14B3843A3', '2020-12-07 20:45:21.986389+00', '2021-03-01');
```

Comment:
_note, not a blocker_
This looks like a place where a database-generated default for the UUID and timestamp might be useful -- we could seed the relevant fields without having to pre-generate values for the plumbing.

Reply:
agree. but having the abililty to add records with known uuids, or created dates in the past is a benefit.

Reply:
true